Calculators/BMI-Calculator/BMI-Calculator.html

```diff
@@ -2,32 +2,59 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" type="text/css" href="BMI-Calculator.css" />
-    <title>BMI Calculator - CalcDiverse</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
```

Comment:
Pls change the file name here @VardaPanchal005

Reply:
oo okay

Reply:
I changed it will it get updated ?

Reply:
Yes, updated